cmd/explaintest/t/topn_push_down.test

```diff
@@ -121,10 +121,10 @@ CREATE TABLE `te` (
   `device_identy` varchar(36) NOT NULL,
   `uuid` varchar(32) NOT NULL,
   `status_flag` tinyint(4) NOT NULL,
-  `client_create_time` timestamp(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `client_create_time` timestamp(3) NOT NULL DEFAULT CURRENT_TIMESTAMP(3),
```

Comment:
Can we not adding `(3)` to `CURRENT_TIMESTAMP`?

Reply:
check fsp in create table is rollbacked in this PR, 83b4e233457a98dd5f52d633a44445d63b351e08.